internal/database/client.go

```diff
@@ -0,0 +1,50 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/boltdb/bolt"
+)
+
+const (
+	databaseFile = "madprobe.db"
```

Comment:
Make this thing configurable. You'll find an example [here].(https://github.com/MadJlzz/madprobe/blob/master/internal/config.go)
Maybe create a new BoltDBConfiguration with some defaults and use `viper` to retrieve value through CLI, property file, environment variable ? What do you think ?

Reply:
Yes it would be much better

Reply:
I added the database file configuration to this file in order to keep configuration in one place. Is it ok like this ?

The new function GetDatabaseFile is public unlike the others, is it an issue ? I did it so we don't have to handler ServerConfiguration every where, it is safe to call GetDatabaseFile alone as ServerConfiguration should already have been initiated in the main.

Reply:
Yep this is fine for me.